corporate/finance/bookkeeping-standards.rst

```diff
@@ -40,6 +40,11 @@ Students must `submit reimbursement requests in Ramp <https://support.ramp.com/h
 
 Once the request is approved by a department owner and an administrator, :ref:`Ramp <corporate/key-vendors:Ramp>` will withdraw the funds from our `Ramp Business Account <https://support.ramp.com/hc/en-us/articles/35043351807507-Ramp-Business-Account-Overview>`_.
 
+Accidental transactions
+-----------------------
+In the event a Ramp card is used for a personal transaction, the transaction must be `repaid by the card holder within Ramp<https://support.ramp.com/hc/en-us/articles/1500008385922-Accidental-or-personal-charges-on-Ramp>`_.
+When synchronizing an accidental transaction to QuickBooks, categorize it under **99999 - Accidental and Mischarge**.
```

Comment:
Vale is throwing a spellcheck error here - you can either add "Mischarge" to the [accepted vocabulary](https://github.com/RoboJackets/business-process-manual/blob/main/.styles/config/vocabularies/RoboJackets/accept.txt) or add 
```
.. vale Vale.Spelling = NO
```
 above this paragraph.